WordPress/src/androidTest/java/org/wordpress/android/e2e/LoginTests.java

```diff
@@ -32,7 +32,7 @@ public void setUp() {
 
     @Test
     public void loginWithEmailPassword() {
-        wpLogin();
+        new LoginFlow().loginEmailPassword();
```

Comment:
Why has this been changed?

Reply:
Changed the default login to the site address flow because of https://github.com/wordpress-mobile/WordPress-Android/issues/7922, alternatively, with the implemented fix I could have handled the error and then enter the username/password when prompted, I think I'll add that for this specific test but leave the `wpLogin` to use site address, WDYT?

Reply:
Oh, I see. What you have done makes sense.

Do all of the login tests pass with these changes?

Reply:
Currently this one `loginWithEmailPassword()` has a chance of failing because of the above-mentioned error, I'm wondering if it makes sense to update it to follow the error and then use the username and password but doing that doesn't really make it an email/password test anymore

Reply:
Actually, thinking about this again, are these changes needed at all? Since the tests are fully mocked, we will never encounter the API error. What do you think?

Reply:
Interesting, I never thought about that but in any case, this change doesn't affect that it just changed the default login to the site address flow, this test should still pass since the tests are fully mocked.

Reply:
Right, but do we need to change to the site address flow since we will not be affected by https://github.com/wordpress-mobile/WordPress-Android/issues/7922?

Reply:
The site address flow isn't what would've been changed it would be the email/password flow that would be updated to handle the error and I hadn't updated it yet so it should be good to go!

Reply:
Yep! tested it out again and it works fine 👍

Reply:
@jtreanor Since we're mocking these well now there's no need for the login tests to change, I've gone ahead and reverted the related changes